webapi/src/main/scala/org/knora/webapi/routing/RouteUtilV2.scala

```diff
@@ -176,24 +204,52 @@ object RouteUtilV2 {
    * @param requestContext the request context.
    * @return the set of schema options submitted in the request, including default options.
    */
-  def getSchemaOptions(requestContext: RequestContext): Set[SchemaOption] =
-    Set(
-      getStandoffRendering(requestContext),
-      getJsonLDRendering(requestContext)
-    ).flatten
+  @deprecated("use getSchemaOptions() instead")
+  def getSchemaOptionsUnsafe(requestContext: RequestContext): Set[SchemaOption] =
+    getSchemaOptions(requestContext).fold(
+      errors => throw errors.head,
+      schemaOptions => schemaOptions
+    )
+
+  /**
+   * Gets the schema options submitted in the request.
+   *
+   * @param requestContext the request context.
+   * @return the set of schema options submitted in the request, including default options.
+   */
+  def getSchemaOptions(requestContext: RequestContext): Validation[Throwable, Set[SchemaOption]] =
```

Comment:
```suggestion
  def getSchemaOptions(requestContext: RequestContext): Validation[BadRequestException, Set[SchemaOption]] =
```
After changing the functions `getStandoffRendering` and `getJsonLDRendering` this type can probably also be narrowed.